docs/usage-as-end-user.md

```diff
@@ -137,15 +137,23 @@ apiVersion: openstack.provider.extensions.gardener.cloud/v1alpha1
 kind: WorkerConfig
 serverGroup:
   policy: soft-anti-affinity
+nodeTemplate: # (to be specified only if the node capacity would be different from cloudprofile info during runtime)
+  capacity:
+    cpu: 2
+    gpu: 0
+    memory: 50Gi
 ```
 
+### ServerGroups
 When you specify the `serverGroup` section in your worker group configuration, a new server group will be created with the configured policy for each worker group that enabled this setting and all machines managed by this worker group will be assigned as members of the created server group.
 
 For users to have access to the server group feature, it must be enabled on the `CloudProfile` by your operator. 
 Existing clusters can take advantage of this feature by updating the server group configuration of their respective worker groups. Worker groups that are already configured with server groups can update their setting to change the policy used, or remove it altogether at any time.
 
 Users must be aware that **any change to the server group settings will result in a rolling deployment of new nodes for the affected worker group**.
 
+### Node Templates
```

Comment:
Section as such looks good, but should probably be placed after the next paragraph (which is still on the topic of server groups)

Reply:
fixed. That was silly of me